ui/src/@types.tsx

```diff
@@ -32,9 +33,16 @@ export type RepoDataPropsT = {
 }
 
 /** Repository Data Props Type */
+export type RepoSyncData = {
+  name: string
+  type: RepoType
+  syncs?: Array<RepoSyncDataType>
+}
+
 export type SyncStatusDataT = {
   value: number
```

Comment:
I think it might be good to rename this to `runningTime` to be clearer about what type of data is held here. It is a duration as well, if we use seconds, maybe it should be `runningTimeSeconds` and that way in downstream components we can display it in different ways if needed (`4 min` `1 hour` etc)

Reply:
I left `runningTime` to be more generic, maybe we could need to manage mins later. Done [0915532](https://github.com/mergestat/fuse/pull/89/commits/091553222b1970608f9d6df69106117f0a092535)